google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineSupportProvider.java

```diff
@@ -150,9 +146,10 @@ private void addSupport(final Module module,
     }
 
     final Project project = module.getProject();
-    webIntegration.addDevServerToModuleDependencies(rootModel, sdk);
+    webIntegration.addDevServerToModuleDependencies(rootModel);
 
-    final Library apiJar = addProjectLibrary(module, "AppEngine API", sdk.getUserLibraryPaths(),
+    final Library apiJar = addProjectLibrary(module, "AppEngine API",
```

Comment:
Yes, but I believe the trigger is adding the framework support (which in turn creates the facet).